addons/full-signature/working.js

```diff
@@ -0,0 +1,32 @@
+export default async function ({ addon, global, console }) {
+  document.querySelector(".activity-stream").insertAdjacentHTML(
+    "beforeend",
+    `
+  <button class="load-more-wibd">Load More</button>
+  `
+  );
+  let dataLoaded = 6;
+  let loadMore = document.querySelector(".load-more-wibd");
+  loadMore.addEventListener("click", function () {
+    fetch(
+      `https://scratch.mit.edu/messages/ajax/user-activity/?user=${
+        document.querySelector(".header-text h2").innerHTML
+      }&max=1000000`
+    )
+      .then(function (response) {
+        return response.text();
+      })
+      .then(function (text) {
+        let dummyEl = document.createElement("div");
+        dummyEl.innerHTML = text;
+        let lastDataLoad = dataLoaded;
+        for (; dataLoaded < lastDataLoad + 6; dataLoaded++) {
```

Comment:
lol I've never seen a for with a semicolon at the beginning, but I know it works

Reply:
Without it, it gives an error